src/protobuf-net.Reflection/CodeGenerator.cs

```diff
@@ -143,7 +143,7 @@ public virtual string GetAccess(Access access)
         /// <summary>
         /// Execute the code generator against a FileDescriptorSet, yielding a sequence of files
         /// </summary>
-        public override IEnumerable<CodeFile> Generate(FileDescriptorSet set, NameNormalizer normalizer = null, Dictionary<string, string> options = null)
+        public override IEnumerable<CodeFile> Generate(FileDescriptorSet set, NameNormalizer normalizer = null, Dictionary<string, string> options = null, Dictionary<string, string> customNamesConversion = null)
```

Comment:
here we are fine to just add the param - as the `override` is just going to override the more specific one, the one that will get called *anyway*.